src/components/AppMenu.tsx

```diff
@@ -46,6 +46,7 @@ export function AppMenu() {
           Sign Out
         </MenuItem>
         <Divider />
+        <MenuItem component={Link} href='/about'>About</MenuItem>
```

Comment:
its using `Link` from `@mui/material` is there a difference?

Reply:
It affects the styling. I've implemented it on the AppMenu because I think that uses the routing, but I reverted it on the about page because it affected styling and those links are not routing in the app -- they are either on the page, or external.

Reply:
if not... then maybe we need to compose them? https://dev.to/ivandotv/using-next-js-link-component-with-material-ui-buttons-and-menu-items-3m6a

Reply:
The article seems to be good info. I don't know that the app is doing the right thing elsewhere. I think there's a follow-up PR to fix the rest of the app. I don't care if you push this before that with or without the correct implementation.